lib/bandit.ex

```diff
@@ -275,9 +278,16 @@ defmodule Bandit do
     |> ThousandIsland.start_link()
     |> case do
       {:ok, pid} ->
-        Logger.info(info(scheme, display_plug, pid))
+        startup_log && Logger.log(startup_log, info(scheme, display_plug, pid))
         {:ok, pid}
 
+      {:error, {:shutdown, {:failed_to_start_child, :listener, :eaddrinuse}}} = error ->
+        port = Keyword.get(options, :port, 4000)
+        address = options |> Keyword.get(:transport_options, []) |> Keyword.get(:ip, {127, 0, 0, 1})
```

Comment:
This is expressing policy that's part of Thousand Island (specifically, default ports & bind addresses). We should really be pulling this out of config somehow - let me tinker with this and see what I can come up with

Reply:
Yup, didn't find a better way to pull this info, so just guessing here from the opts. The Cowboy2 adapter in Phoenix uses the ranch ref to get info from a local ETS table. I don't know if there's any way to get the info from Bandit once the supervision tree has been terminated? Feel free to continue work on this PR!

Reply:
> Again it's all good with ability to silence the info log, just some thoughts on why it should be there in the first place

Yep, totally agree - I think the config point we're adding here handles this well. I really *do* like having the logging by default for all Bandit installs; I personally use it all the time just to double check myself.